lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -137,6 +137,79 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
                       binder.op, resultType, lhs, rhs);
                   return success();
                 });
+  patterns.onOp(
+      "MatMulInteger", 10,
+      [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        llvm::SmallVector<Value> operands;
+        if (binder.tensorOperands(operands, 4) ||
+            binder.tensorResultType(resultType))
+          return failure();
+
+        Value lhs = operands[0];
+        Value rhs = operands[1];
+        Value lhsZp = operands[2];
+        Value rhsZp = operands[3];
```

Comment:
The `a_zero_point` and `b_zero_point` are optional args, they might not be present always.

Reply:
And, indexing this would result in a crash.

Reply:
Changed to conditionally pull the values (see above). Do we know if the optional representation is via a `None` value or absect operands? I had run on the assumption that `onnx` used none types for missing parameters.

Reply:
No, the way onnx operators are imported the optional value is either present or absent.